### DISCUSSION TITLE
Welcome to MSA5_TEAM3 Discussions!

### DISCUSSION THREAD

**Discussion by webvocado** [webvocado/MSA5_TEAM3#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ouDangTang** [comment]
1따봉ㅇ드립니다

**Reply by webvocado** [reply]
개이득

**Reply by ouDangTang** [reply]
화살표 뭔지 알아요 ?

**Reply by webvocado** [reply]
모르겠어용

**Comment by Dozzykim** [comment]
붐업
